weblab-docker/Dockerfile

```diff
@@ -16,13 +16,20 @@ RUN apt update \
 
 SHELL ["/bin/bash", "-c"]
 
-# Copy the full repo to the docker image
 ARG ANDY_DIR=/home/andy/
 ARG ANDY_BRANCH=main
-RUN git clone -n https://github.com/cse1110/andy.git && git -C ${ANDY_DIR} checkout ${ANDY_BRANCH}
```

Comment:
Can’t we simply pass a dynamic variable here, like the latest_commit one you use? Git clone accepts a -c option, allowing you to pass a configuration. Would that simplify?

Reply:
What do you mean exactly? What would this configuration contain?

Reply:
Forget it. I thought “latest commit” was a variable, but this is simply putting the content of the page to this file. Can’t we just use some docker flag to force it to ignore the cache and rebuild the image?

Reply:
There doesn't seem to be anything like that, unfortunately... We could clear the entire cache but this would affect the cache of other images unrelated to Andy too, so it's not ideal.

Reply:
That sorts this discussion out then!